.env

```diff
@@ -21,6 +21,17 @@ REDIS_IMAGE=redis
 # App image name for use if you intend to push it to docker registry/hub.
 APP_PROD_IMAGE=my-ez-app
 APP_DOCKER_FILE=Dockerfile
+NGINX_PROD_IMAGE=my-ez-web
+
+NGINX_DOCKER_FILE=Dockerfile.nginx
+
+DATASET_DBDUMP_DOCKER_FILE=Dockerfile.dbdump
+DATASET_DBDUMP_IMAGE=my-ez-app-dataset-dbdump
+
+DATASET_VARDIRDUMP_DOCKER_FILE=Dockerfile.vardirdump
+DATASET_VARDIRDUMP_IMAGE=my-ez-app-dataset-vardirdump
+
+DATASET_VARDIR=my-ez-app
 
 # Install config, used by .platform.app.yaml among others
-INSTALL_EZ_INSTALL_TYPE=studio-clean
+INSTALL_EZ_INSTALL_TYPE=demo
```

Comment:
Moved to separate PR https://github.com/ezsystems/ezplatform-ee-demo/pull/35

Will remove comitt from this branch once PR #35 is merged to master